docs/05-concepts/02-models.md

```diff
@@ -159,3 +159,51 @@ extension MyExtension on MyClass {
 |[**fields (index)**](database/indexing)|List the fields to create the indexes on.                                                |✅|||
 |[**type (index)**](database/indexing)|The type of index to create.                                                               |✅|||
 |[**unique**](database/indexing)|Boolean flag to make the entries unique in the database.                                         |✅|||
+
+## Default Values
```

Comment:
Suggestion: Add a simple example demonstrating how to configure a default value to the introduction.

This is most likely what most people are looking for.